doc/radosgw/rgw-cache.rst

```diff
@@ -0,0 +1,98 @@
+========================== 
+RGW Data caching and CDN
+==========================
+
+.. versionadded:: Octopus
+
+.. contents::
+
+This new feature adds to RGW the ability to securely cache objects and offload the workload from the cluster, using Nginx.
+After an object is accessed the first time it will be stored on top of Nginx dir.
+Every request the Nginx will not fetch data from the RGW or the cluster, It will only check for permissions for that user from the RGW.
```

Comment:
Should this say, "When data is already cached, it need not be fetched from RGW.  A permission check will be made against RGW to ensure the requesting user has access."